c/datasetService.c

```diff
@@ -99,13 +99,16 @@ static int serveDatasetContents(HttpService *service, HttpResponse *response){
     respondWithDataset(response, filename, TRUE);
   }
   else if (!strcmp(request->method, methodPOST)){
+    /* this section is WIP to allow saving of members when another user
+       has the directory open */
     char *l1 = stringListPrint(request->parsedFile, 1, 1, "/", 0);
     char *percentDecoded = cleanURLParamValue(response->slh, l1);
-    char *filenamep1 = stringConcatenate(response->slh, "//'", percentDecoded);
-    char *filename = stringConcatenate(response->slh, filenamep1, "'");
+    char *filenamep1 = stringConcatenate(response->slh, "//'", percentDecoded);     /* prepend  //'  to DSN*/
+    char *filename = stringConcatenate(response->slh, filenamep1, "'");             /* append   '    to DSN */
+
     zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG, "Updating if exists: %s\n", filename);
     fflush(stdout);
-    updateDataset(response, filename, TRUE);
+    updateDataset(response, filename, TRUE);   
```

Comment:
Don't add trailing space.

Reply:
It's harmless in C.  If there's a formatting standard, then we should have a code formatter to fix this sort of thing.  Saves time.

Reply:
It's a change that does nothing. We shouldn't commit trailing spaces.

The rule of thumb is to use the same style as the rest of the file.